electron-builder.yml

```diff
@@ -35,6 +41,15 @@ linux:
     - rpm
     - snap
 
+rpm:
+  fpm:
+    - "--after-install=assets/build/after-install.sh"
+    - "--after-remove=assets/build/after-remove.sh"
+
+deb:
+  afterInstall: assets/build/after-install.sh
+  afterRemove: assets/build/after-remove.sh
+
 snap:
```

Comment:
How does it work for `snap` installs?

Reply:
As mentioned above, it doesn't. Only on `deb` for now.